cmd/tailscaled/tailscaled.go

```diff
@@ -96,6 +97,70 @@ var subCommands = map[string]*func([]string) error{
 	"debug":                   &debugModeFunc,
 }
 
+// tryWindowsAppDataMigration attempts to move the Windows state storage
+// directory from its old location to the new location. (Issue 2856)
+//
+// Tailscale 1.14 and before stored state under %LocalAppData%
+// (usually "C:\WINDOWS\system32\config\systemprofile\AppData\Local"
+// when tailscaled.exe is running as a non-user system service).
+// However it is frequently cleared for almost any reason: Windows
+// updates, System Restore, even various System Cleaner utilities.
+//
+// This function does not return an error; on rename failure, code elsewhere
```

Comment:
not "rename" anymore

maybe just "on failure"

Reply:
Done